content/raceday.inc

```diff
@@ -7,20 +7,24 @@
       <a class="nav-link" href="/raceday/heats">Heat Schedule</a>
     </li>
     <li class="nav-item">
-      <a class="nav-link" href="/history">Leaderboard</a>
+      <a class="nav-link" href="/history/rosters/2022">Rosters</a>
     </li>
     <li class="nav-item">
-      <a class="nav-link" target="_blank" rel="noopener" href="https://discord.gg/PUwuUuj">Chat</a>
+      <a class="nav-link" href="/raceday/leaderboard">Leaderboard</a>
     </li>
     <li class="nav-item">
-      <a class="nav-link" href="/live">Live</a>
+      <a class="nav-link" target="_blank" rel="noopener" href="/chat">Chat</a>
+    </li>
+    <li class="nav-item">
+      <a class="nav-link" href="/raceday/livestream">Livestream</a>
     </li>
   </ul>
 </nav>
 <?php
-  $page = "overview";
   if(isset($_GET['p'])){
     $page = $_GET['p'];
+  } else {
+    die('It looks like nothing was found at this location. Maybe try one of the links on the side?');
   }
   include_once("content/raceday/".$page.".inc");
```

Comment:
Not your fault,. but this should probably check for the existance of the file like index.php does, before using it (and it can possibly make use of 404.inc in the same way).